src/search_algo.hpp

```diff
@@ -367,7 +367,26 @@ loadQuery(GlobalDataHolder<c_indexType, c_origSbjAlph, c_transAlph, c_redAlph, c
         throw QueryException{"Zero sequences submitted."};
     }
 
-    // TODO if constexpr BLASTN mode, reverse complements
+    if constexpr (TGH::blastProgram == seqan::BlastProgram::BLASTN)
+    {
+        //TODO view-ify this at some point. Better yet, move to index.
```

Comment:
Why move to index at some point?

Reply:
Well, copying and revcomplementing doubles the number of query sequences which doubles the run-time of the program (linear influence).

 Reverse-complementing instead the database sequences only has logarithmic influence on the run-time.
OTOH it doubles the required memory which is also very significant.

It's a difficult call.